crashy/README.md

```diff
@@ -0,0 +1,25 @@
+# Crashy
```

Comment:
One of the things we've been trying to do with each of the samples is give people a little head start by listing goals for the sample and the important files. The important files for this one would really just be main.dart and the missing dsn.dart, and the goals would just be to understand how using a zone can help you trap exceptions and report on them.

You can see the jsonexample's README for an example of how this works elsewhere:

https://github.com/flutter/samples/tree/master/jsonexample